src/test/regress/expected/multi_extension.out

```diff
@@ -555,7 +555,9 @@ ALTER EXTENSION citus UPDATE TO '10.0-3';
 SELECT * FROM print_extension_changes();
  previous_object | current_object
 ---------------------------------------------------------------------
-(0 rows)
+                 | function citus_internal.downgrade_columnar_storage(regclass)
+                 | function citus_internal.upgrade_columnar_storage(regclass)
+(2 rows)
```

Comment:
This diff still doesn't seem to be correct. Let me share how should it look like via a commit on a separate branch:
a8d8953b831ce40290b9db187217387b2baa0382

Reply:
Sorry, I missed that one. I included your commit, thank you.